tests/integration/cli/up_test.py

```diff
@@ -9,6 +10,13 @@ def setUp(self):
         busybox_single_app_bundle_fixture(num_bundles=2)
         self.run_command('bundles activate busyboxa busyboxb')
 
+    def tearDown(self):
+        try:
+            self.run_command('stop')
+        except Exception:
+            pass
+        super(TestUpCLI, self).tearDown()
```

Comment:
When we explicitly stop we're able to provide a timeout value on the SIGTERM. We pass this as 1 second, down from the default of 10 seconds.

If you do a `docker-compose up` and there are old versions of the containers running, it'll stop them for you with the 10 second default.